src/structures/GuildMember.js

```diff
@@ -80,6 +86,9 @@ class GuildMember extends Base {
       this.premiumSinceTimestamp = data.premium_since ? new Date(data.premium_since).getTime() : null;
     }
     if ('roles' in data) this._roles = data.roles;
+    if('communication_disabled_until' in data) {
+      this.communicationDisabledUntilTimestamp = data.communication_disabled_until ? new Date(data.communication_disabled_until).getTime() : null;
```

Comment:
you can assume that data.communication_disabled_until is always truthy here

Reply:
no because it can also be null which would be falsy (when referring to the ternary)
also i dont know if it exists on a partial member so i put it in just to be sure